src/Consumer/DoiConsumer.php

```diff
@@ -123,6 +123,15 @@ public function execute(AMQPMessage $message)
             } else {
                 $this->logger->warning("Unknown routing key: $routingKey", $loggingContext);
             }
+
+            if (preg_match('/^doi/', $routingKey)) {
```

Comment:
Remove above issue match, because it's throwing a warning of bad routing key. It works fine however.

Reply:
Fixed it